course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -360,7 +360,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(cubes.extract('air_potential_temperature' & \n",
+    "print(cubes.extract('air_temperature' & \n",
```

Comment:
Likewise, correct as-is

Reply:
Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)